OLE-objects/Get-file-path-of-linked-OLE-Object/.NET/Get-file-path-of-linked-OLE-Object/Get-file-path-of-linked-OLE-Object.csproj

```diff
@@ -12,4 +12,16 @@
     <PackageReference Include="Syncfusion.Presentation.NET" Version="*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Data\Template.pptx">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+	<None Update="Data\HelloWorld.docx">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>

```

Comment:
whether this HelloWorld document used in the cs file

Reply:
Yes, @LokeshBaskar. This HelloWorld.docx is used in the cs file.